src/applications/personalization/dashboard/components/health-care-v2/HealthCareContentV2.jsx

```diff
@@ -27,7 +27,7 @@ import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
 import Appointments from './AppointmentsV2';
 import CTALink from '../CTALink';
 
-const HealthCareContentV2 = ({
+export const HealthCareContentV2 = ({
```

Comment:
I'm somewhat torn with exporting this named const, since we have the default export at the bottom and the default is imported as HealthCareContentV2 in HealthCareV2. That has bit me in the butt before when I thought I was importing one thing and accidentally imported the unconnected version resulting in errors.

Maybe you could do this at the bottom instead:
```
export const UnconnectedHealthCareContentV2 = HealthCareContentV2;
```